11/22/2022 20:15:37 Application Starting Up on "Tuesday, November 22, 2022" at "7:15:37 PM"
11/22/2022 20:15:45 User requested the list of CurvePoints on "Tuesday, November 22, 2022" at "7:15:45 PM"
11/22/2022 20:15:56 User requested the list of CurvePoints on "Tuesday, November 22, 2022" at "7:15:56 PM"
11/22/2022 20:16:07 User requested the BidList with Id = 1 on "Tuesday, November 22, 2022" at "7:16:07 PM"
11/22/2022 20:16:16 User requested the BidList with Id = 1023 on "Tuesday, November 22, 2022" at "7:16:16 PM"
11/22/2022 20:19:03 User requested the CurvePoint with Id = 2 on "Tuesday, November 22, 2022" at "7:19:03 PM"
11/22/2022 20:21:42 User created a new CurvePoint on "Tuesday, November 22, 2022" at "7:21:42 PM"
11/24/2022 19:58:54 Application Starting Up on "Thursday, November 24, 2022" at "6:58:54 PM"
11/24/2022 19:59:20 User requested the list of Ratings on "Thursday, November 24, 2022" at "6:59:20 PM"
11/24/2022 19:59:37 User requested the Rating with Id = 2 on "Thursday, November 24, 2022" at "6:59:37 PM"
11/24/2022 20:00:06 User requested the Rating with Id = 1 on "Thursday, November 24, 2022" at "7:00:06 PM"
11/24/2022 20:02:35 User created a new Rating on "Thursday, November 24, 2022" at "7:02:35 PM"
11/24/2022 20:05:03 User updated the Rating with Id = 2002 on "Thursday, November 24, 2022" at "7:05:03 PM"
11/24/2022 20:06:46 User deleted the Rating with Id = 1002 on "Thursday, November 24, 2022" at "7:06:46 PM"
11/25/2022 10:25:57 Application Starting Up on "Friday, November 25, 2022" at "9:25:57 AM"
11/25/2022 10:26:07 User requested the list of CurvePoints on "Friday, November 25, 2022" at "9:26:07 AM"
11/25/2022 10:41:36 Application Starting Up on "Friday, November 25, 2022" at "9:41:36 AM"
11/25/2022 10:41:55 User requested the Rating with Id = 16 on "Friday, November 25, 2022" at "9:41:55 AM"
11/25/2022 10:45:47 Application Starting Up on "Friday, November 25, 2022" at "9:45:47 AM"
11/25/2022 10:46:07 User requested the Rating with Id = 18 on "Friday, November 25, 2022" at "9:46:07 AM"
11/25/2022 10:46:07 No Rating with Id = 18 was found. User was advised to enter a valis Id.
11/25/2022 10:49:27 Application Starting Up on "Friday, November 25, 2022" at "9:49:27 AM"
11/25/2022 10:49:37 User requested the Rating with Id = 20 on "Friday, November 25, 2022" at "9:49:37 AM"
11/25/2022 10:49:37 No Rating with Id = 20 was found. User was advised to enter a valid Id.
11/25/2022 10:51:11 User requested the list of Ratings on "Friday, November 25, 2022" at "9:51:11 AM"
11/25/2022 11:14:27 Application Starting Up on "Friday, November 25, 2022" at "10:14:27 AM"
11/25/2022 11:17:49 Application Starting Up on "Friday, November 25, 2022" at "10:17:49 AM"
11/25/2022 11:22:53 Application Starting Up on "Friday, November 25, 2022" at "10:22:53 AM"
11/25/2022 11:28:26 Application Starting Up on "Friday, November 25, 2022" at "10:28:26 AM"
11/25/2022 11:29:36 User created a new Rating on "Friday, November 25, 2022" at "10:29:36 AM"
11/25/2022 11:33:15 User updated the Rating with Id = 3002 on "Friday, November 25, 2022" at "10:33:15 AM"
11/25/2022 11:35:05 User deleted the Rating with Id = 3002 on "Friday, November 25, 2022" at "10:35:05 AM"
11/25/2022 11:43:49 Application Starting Up on "Friday, November 25, 2022" at "10:43:49 AM"
11/25/2022 11:44:10 User requested to delete the Rating with Id = 3010 on "Friday, November 25, 2022" at "10:44:10 AM"
11/25/2022 11:44:10 No Rating with Id = 3010 was found. User was advised to enter a valid Id.
11/25/2022 11:55:47 Application Starting Up on "Friday, November 25, 2022" at "10:55:47 AM"
11/25/2022 11:57:26 User created a new Rating on "Friday, November 25, 2022" at "10:57:26 AM"
11/25/2022 11:58:22 User updated the Rating with Id = 3003 on "Friday, November 25, 2022" at "10:58:22 AM"
11/25/2022 11:58:49 User requested to delete the Rating with Id = 3004 on "Friday, November 25, 2022" at "10:58:49 AM"
11/25/2022 11:58:49 No Rating with Id = 3004 was found. User was advised to enter a valid Id.
11/25/2022 11:59:25 User requested to update the Rating with Id = 3004 on "Friday, November 25, 2022" at "10:59:25 AM"
11/25/2022 11:59:25 No Rating with Id = 3004 was found. User was advised to enter a valid Id.
11/25/2022 11:59:55 User deleted the Rating with Id = 3003 on "Friday, November 25, 2022" at "10:59:55 AM"
11/25/2022 12:11:19 Application Starting Up on "Friday, November 25, 2022" at "11:11:19 AM"
11/25/2022 14:53:40 Application Starting Up on "Friday, November 25, 2022" at "1:53:40 PM"
11/25/2022 14:53:55 User requested the list of Ratings on "Friday, November 25, 2022" at "1:53:55 PM"
11/25/2022 14:54:21 User requested the Rating with Id = 1 on "Friday, November 25, 2022" at "1:54:21 PM"
11/25/2022 15:02:49 User requested the Rating with Id = 101 on "Friday, November 25, 2022" at "2:02:49 PM"
11/25/2022 15:02:50 No Rating with Id = 101 was found. User was advised to enter a valid Id.
11/25/2022 15:17:57 Application Starting Up on "Friday, November 25, 2022" at "2:17:57 PM"
11/25/2022 15:18:25 User created a new Rating on "Friday, November 25, 2022" at "2:18:25 PM"
11/25/2022 15:23:49 Application Starting Up on "Friday, November 25, 2022" at "2:23:49 PM"
11/25/2022 15:24:04 User created a new Rating on "Friday, November 25, 2022" at "2:24:04 PM"
11/25/2022 15:25:06 Application Starting Up on "Friday, November 25, 2022" at "2:25:06 PM"
11/25/2022 15:29:53 Application Starting Up on "Friday, November 25, 2022" at "2:29:53 PM"
11/25/2022 15:30:02 HTTP "POST" "/rating/add" responded 500 in 28.2731 ms
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Services.RatingService' while attempting to activate 'Dot.Net.WebApi.Controllers.RatingController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
11/25/2022 15:30:02 An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Services.RatingService' while attempting to activate 'Dot.Net.WebApi.Controllers.RatingController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
11/25/2022 15:34:00 Application Starting Up on "Friday, November 25, 2022" at "2:34:00 PM"
11/25/2022 15:34:00 The Application Failed to Start Correctly.
System.AggregateException: Some services are not able to be constructed (Error while validating the service descriptor 'ServiceType: WebApi3.Services.RatingService Lifetime: Scoped ImplementationType: WebApi3.Services.RatingService': Unable to resolve service for type 'WebApi3.Repositories.IRatingRepository' while attempting to activate 'WebApi3.Services.RatingService'.)
 ---> System.InvalidOperationException: Error while validating the service descriptor 'ServiceType: WebApi3.Services.RatingService Lifetime: Scoped ImplementationType: WebApi3.Services.RatingService': Unable to resolve service for type 'WebApi3.Repositories.IRatingRepository' while attempting to activate 'WebApi3.Services.RatingService'.
 ---> System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.IRatingRepository' while attempting to activate 'WebApi3.Services.RatingService'.
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteFactory.CreateArgumentCallSites(Type serviceType, Type implementationType, CallSiteChain callSiteChain, ParameterInfo[] parameters, Boolean throwIfCallSiteNotFound)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteFactory.CreateConstructorCallSite(ResultCache lifetime, Type serviceType, Type implementationType, CallSiteChain callSiteChain)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteFactory.TryCreateExact(ServiceDescriptor descriptor, Type serviceType, CallSiteChain callSiteChain, Int32 slot)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteFactory.GetCallSite(ServiceDescriptor serviceDescriptor, CallSiteChain callSiteChain)
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.ServiceProviderEngine.ValidateService(ServiceDescriptor descriptor)
   --- End of inner exception stack trace ---
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.ServiceProviderEngine.ValidateService(ServiceDescriptor descriptor)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider..ctor(IEnumerable`1 serviceDescriptors, ServiceProviderOptions options)
   --- End of inner exception stack trace ---
   at Microsoft.Extensions.DependencyInjection.ServiceProvider..ctor(IEnumerable`1 serviceDescriptors, ServiceProviderOptions options)
   at Microsoft.Extensions.DependencyInjection.ServiceCollectionContainerBuilderExtensions.BuildServiceProvider(IServiceCollection services, ServiceProviderOptions options)
   at Microsoft.Extensions.DependencyInjection.DefaultServiceProviderFactory.CreateServiceProvider(IServiceCollection containerBuilder)
   at Microsoft.Extensions.Hosting.Internal.ServiceFactoryAdapter`1.CreateServiceProvider(Object containerBuilder)
   at Microsoft.Extensions.Hosting.HostBuilder.CreateServiceProvider()
   at Microsoft.Extensions.Hosting.HostBuilder.Build()
   at Dot.Net.WebApi.Program.Main(String[] args) in C:\Users\Gaetan\source\repos\DotNetEnglishP7_ter\src\Program.cs:line 28
11/25/2022 15:37:24 Application Starting Up on "Friday, November 25, 2022" at "2:37:24 PM"
11/25/2022 15:37:33 HTTP "POST" "/rating/add" responded 500 in 30.4350 ms
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Services.RatingService' while attempting to activate 'Dot.Net.WebApi.Controllers.RatingController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
11/25/2022 15:37:34 An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Services.RatingService' while attempting to activate 'Dot.Net.WebApi.Controllers.RatingController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
11/25/2022 15:38:01 Application Starting Up on "Friday, November 25, 2022" at "2:38:01 PM"
11/25/2022 15:38:24 User created a new Rating on "Friday, November 25, 2022" at "2:38:24 PM"
11/25/2022 15:38:52 User requested to delete the Rating with Id = 30010 on "Friday, November 25, 2022" at "2:38:52 PM"
11/25/2022 15:38:52 No Rating with Id = 30010 was found. User was advised to enter a valid Id.
11/25/2022 15:39:03 User deleted the Rating with Id = 3007 on "Friday, November 25, 2022" at "2:39:03 PM"
11/26/2022 11:19:34 Application Starting Up on "Saturday, November 26, 2022" at "10:19:34 AM"
11/26/2022 11:21:45 Application Starting Up on "Saturday, November 26, 2022" at "10:21:45 AM"
11/26/2022 11:21:58 User requested the list of Ratings on "Saturday, November 26, 2022" at "10:21:58 AM"
11/26/2022 11:22:53 User created a new Rating on "Saturday, November 26, 2022" at "10:22:53 AM"
11/26/2022 11:27:44 Application Starting Up on "Saturday, November 26, 2022" at "10:27:44 AM"
11/26/2022 11:27:57 User requested the list of Ratings on "Saturday, November 26, 2022" at "10:27:57 AM"
11/26/2022 11:28:25 User requested the Rating with Id = 500 on "Saturday, November 26, 2022" at "10:28:25 AM"
11/26/2022 11:28:25 No Rating with Id = 500 was found. User was advised to enter a valid Id.
11/26/2022 11:28:42 User requested the Rating with Id = 1 on "Saturday, November 26, 2022" at "10:28:42 AM"
11/26/2022 11:30:55 User created a new Rating on "Saturday, November 26, 2022" at "10:30:55 AM"
11/26/2022 11:33:07 User updated the Rating with Id = 4008 on "Saturday, November 26, 2022" at "10:33:07 AM"
11/26/2022 11:33:48 User requested to delete the Rating with Id = 4010 on "Saturday, November 26, 2022" at "10:33:48 AM"
11/26/2022 11:33:48 No Rating with Id = 4010 was found. User was advised to enter a valid Id.
11/26/2022 11:33:54 User deleted the Rating with Id = 4008 on "Saturday, November 26, 2022" at "10:33:54 AM"
11/26/2022 11:34:01 User deleted the Rating with Id = 4007 on "Saturday, November 26, 2022" at "10:34:01 AM"
11/26/2022 11:34:16 User deleted the Rating with Id = 3004 on "Saturday, November 26, 2022" at "10:34:16 AM"
11/26/2022 11:34:21 User deleted the Rating with Id = 3005 on "Saturday, November 26, 2022" at "10:34:21 AM"
11/26/2022 12:15:19 Application Starting Up on "Saturday, November 26, 2022" at "11:15:19 AM"
11/26/2022 12:15:43 User requested the list of CurvePoints on "Saturday, November 26, 2022" at "11:15:43 AM"
11/26/2022 12:15:51 User requested the BidList with Id = 1 on "Saturday, November 26, 2022" at "11:15:51 AM"
11/26/2022 12:16:04 User requested the BidList with Id = 2 on "Saturday, November 26, 2022" at "11:16:04 AM"
11/26/2022 13:01:33 Application Starting Up on "Saturday, November 26, 2022" at "12:01:33 PM"
11/26/2022 13:01:45 User requested the list of Ratings on "Saturday, November 26, 2022" at "12:01:45 PM"
11/26/2022 13:02:02 User requested the Rating with Id = 1 on "Saturday, November 26, 2022" at "12:02:02 PM"
11/26/2022 13:02:12 User requested the Rating with Id = 11 on "Saturday, November 26, 2022" at "12:02:12 PM"
11/26/2022 13:02:12 No Rating with Id = 11 was found. User was advised to enter a valid Id.
11/26/2022 13:13:16 User created a new Rating on "Saturday, November 26, 2022" at "12:13:16 PM"
11/26/2022 13:14:45 User updated the Rating with Id = 4009 on "Saturday, November 26, 2022" at "12:14:45 PM"
11/26/2022 13:15:14 User requested to delete the Rating with Id = 4010 on "Saturday, November 26, 2022" at "12:15:14 PM"
11/26/2022 13:15:14 No Rating with Id = 4010 was found. User was advised to enter a valid Id.
11/26/2022 13:15:36 User deleted the Rating with Id = 4009 on "Saturday, November 26, 2022" at "12:15:36 PM"
11/26/2022 13:16:22 User requested the list of CurvePoints on "Saturday, November 26, 2022" at "12:16:22 PM"
11/26/2022 13:16:35 User requested the CurvePoint with Id = 1 on "Saturday, November 26, 2022" at "12:16:35 PM"
11/26/2022 13:16:43 User requested the CurvePoint with Id = 10 on "Saturday, November 26, 2022" at "12:16:43 PM"
11/26/2022 13:20:31 Application Starting Up on "Saturday, November 26, 2022" at "12:20:31 PM"
11/26/2022 13:25:16 Application Starting Up on "Saturday, November 26, 2022" at "12:25:16 PM"
11/26/2022 13:25:36 User requested the list of Ratings on "Saturday, November 26, 2022" at "12:25:36 PM"
11/26/2022 13:26:36 Application Starting Up on "Saturday, November 26, 2022" at "12:26:36 PM"
11/26/2022 13:26:47 No Rating with Id = 102 was found. User was advised to enter a valid Id.
11/26/2022 13:28:45 Application Starting Up on "Saturday, November 26, 2022" at "12:28:45 PM"
11/26/2022 13:28:57 User requested the list of Ratings on "Saturday, November 26, 2022" at "12:28:57 PM"
11/26/2022 13:29:07 User requested the Rating with Id = 100 on "Saturday, November 26, 2022" at "12:29:07 PM"
11/26/2022 13:29:07 No Rating with Id = 100 was found. User was advised to enter a valid Id.
11/26/2022 13:29:13 User requested the Rating with Id = 2 on "Saturday, November 26, 2022" at "12:29:13 PM"
11/26/2022 13:29:13 No Rating with Id = 2 was found. User was advised to enter a valid Id.
11/26/2022 13:29:21 User requested the Rating with Id = 1 on "Saturday, November 26, 2022" at "12:29:21 PM"
11/26/2022 13:39:54 Application Starting Up on "Saturday, November 26, 2022" at "12:39:54 PM"
11/26/2022 13:40:02 User requested the list of CurvePoints on "Saturday, November 26, 2022" at "12:40:02 PM"
11/26/2022 13:40:17 User requested the CurvePoint with Id = 1 on "Saturday, November 26, 2022" at "12:40:17 PM"
11/26/2022 13:40:24 User requested the CurvePoint with Id = 11 on "Saturday, November 26, 2022" at "12:40:24 PM"
11/26/2022 13:40:24 No CurvePoint with Id = 11 was found. User was advised to enter a valid Id.
11/26/2022 13:42:25 User created a new CurvePoint on "Saturday, November 26, 2022" at "12:42:25 PM"
11/26/2022 13:44:34 User updated the CurvePoint with Id = 9002 on "Saturday, November 26, 2022" at "12:44:34 PM"
11/26/2022 13:59:33 User requested to delete the CurvePoint with Id = 9010 on "Saturday, November 26, 2022" at "12:59:33 PM"
11/26/2022 13:59:33 No CurvePoint with Id = 9010 was found. User was advised to enter a valid Id.
11/26/2022 13:59:49 User deleted the CurvePoint with Id = 9002 on "Saturday, November 26, 2022" at "12:59:49 PM"
11/26/2022 14:25:59 Application Starting Up on "Saturday, November 26, 2022" at "1:25:59 PM"
11/26/2022 14:26:06 User requested the list of Bids on "Saturday, November 26, 2022" at "1:26:06 PM"
11/26/2022 14:29:21 Application Starting Up on "Saturday, November 26, 2022" at "1:29:21 PM"
11/26/2022 14:29:27 User requested the list of Bids on "Saturday, November 26, 2022" at "1:29:27 PM"
11/26/2022 14:32:10 Application Starting Up on "Saturday, November 26, 2022" at "1:32:10 PM"
11/26/2022 14:32:18 User requested the list of CurvePoints on "Saturday, November 26, 2022" at "1:32:18 PM"
11/26/2022 14:41:41 Application Starting Up on "Saturday, November 26, 2022" at "1:41:41 PM"
11/26/2022 14:41:48 User requested the list of Bids on "Saturday, November 26, 2022" at "1:41:48 PM"
11/26/2022 15:24:37 Application Starting Up on "Saturday, November 26, 2022" at "2:24:37 PM"
11/26/2022 15:24:45 User requested the list of CurvePoints on "Saturday, November 26, 2022" at "2:24:45 PM"
11/26/2022 15:25:06 User requested the BidList with Id = 1 on "Saturday, November 26, 2022" at "2:25:06 PM"
11/26/2022 15:25:06 No Rating with Id = 1 was found. User was advised to enter a valid Id.
11/26/2022 15:25:19 User requested the BidList with Id = 2 on "Saturday, November 26, 2022" at "2:25:19 PM"
11/26/2022 15:26:40 User created a new CurvePoint on "Saturday, November 26, 2022" at "2:26:40 PM"
11/26/2022 15:30:23 User updated the BidList with Id = 3010 on "Saturday, November 26, 2022" at "2:30:23 PM"
11/26/2022 15:35:12 Application Starting Up on "Saturday, November 26, 2022" at "2:35:12 PM"
11/26/2022 15:36:19 User created a new BidList on "Saturday, November 26, 2022" at "2:36:19 PM"
11/26/2022 15:37:29 User requested the list of BidList on "Saturday, November 26, 2022" at "2:37:29 PM"
11/26/2022 15:38:22 User updated the BidList with Id = 3011 on "Saturday, November 26, 2022" at "2:38:22 PM"
11/26/2022 15:38:35 User requested to delete the BidList with Id = 3012 on "Saturday, November 26, 2022" at "2:38:35 PM"
11/26/2022 15:38:35 No BidList with Id = 3012 was found. User was advised to enter a valid Id.
11/26/2022 15:38:40 User deleted the BidList with Id = 3011 on "Saturday, November 26, 2022" at "2:38:40 PM"
11/28/2022 19:30:44 Application Starting Up on "Monday, November 28, 2022" at "6:30:44 PM"
11/28/2022 19:30:57 User requested the list of RuleNames on "Monday, November 28, 2022" at "6:30:57 PM"
11/28/2022 19:31:23 User requested the RuleName with Id = 1 on "Monday, November 28, 2022" at "6:31:23 PM"
11/28/2022 19:31:32 User requested the RuleName with Id = 10 on "Monday, November 28, 2022" at "6:31:32 PM"
11/28/2022 19:31:32 No RuleName with Id = 10 was found. User was advised to enter a valid Id.
11/28/2022 19:34:38 User created a new RuleName on "Monday, November 28, 2022" at "6:34:38 PM"
11/28/2022 19:36:42 User created a new RuleName on "Monday, November 28, 2022" at "6:36:42 PM"
11/28/2022 19:36:59 User created a new RuleName on "Monday, November 28, 2022" at "6:36:59 PM"
11/28/2022 19:44:59 User created a new RuleName on "Monday, November 28, 2022" at "6:44:59 PM"
11/28/2022 19:45:26 User deleted the RuleName with Id = 2003 on "Monday, November 28, 2022" at "6:45:26 PM"
11/28/2022 19:45:35 User deleted the RuleName with Id = 2004 on "Monday, November 28, 2022" at "6:45:35 PM"
11/28/2022 19:45:39 User deleted the RuleName with Id = 2005 on "Monday, November 28, 2022" at "6:45:39 PM"
11/28/2022 19:45:45 User requested to delete the RuleName with Id = 2007 on "Monday, November 28, 2022" at "6:45:45 PM"
11/28/2022 19:45:45 No RuleName with Id = 2007 was found. User was advised to enter a valid Id.
11/28/2022 19:46:40 User updated the RuleName with Id = 2002 on "Monday, November 28, 2022" at "6:46:40 PM"
11/28/2022 19:52:37 Application Starting Up on "Monday, November 28, 2022" at "6:52:37 PM"
11/28/2022 19:52:54 User created a new RuleName on "Monday, November 28, 2022" at "6:52:54 PM"
11/28/2022 19:54:59 User created a new RuleName on "Monday, November 28, 2022" at "6:54:59 PM"
11/28/2022 20:32:11 Application Starting Up on "Monday, November 28, 2022" at "7:32:11 PM"
11/28/2022 20:32:33 User requested the list of RuleNames on "Monday, November 28, 2022" at "7:32:33 PM"
11/28/2022 20:33:47 User updated the RuleName with Id = 2007 on "Monday, November 28, 2022" at "7:33:47 PM"
11/28/2022 20:35:28 Application Starting Up on "Monday, November 28, 2022" at "7:35:28 PM"
11/28/2022 20:36:08 User requested the list of RuleNames on "Monday, November 28, 2022" at "7:36:08 PM"
11/28/2022 20:36:47 User updated the RuleName with Id = 2002 on "Monday, November 28, 2022" at "7:36:47 PM"
11/28/2022 20:37:57 User updated the RuleName with Id = 2006 on "Monday, November 28, 2022" at "7:37:57 PM"
11/28/2022 20:38:09 User deleted the RuleName with Id = 2006 on "Monday, November 28, 2022" at "7:38:09 PM"
11/28/2022 20:38:18 User deleted the RuleName with Id = 2007 on "Monday, November 28, 2022" at "7:38:18 PM"
11/29/2022 20:06:38 Application Starting Up on "Tuesday, November 29, 2022" at "7:06:38 PM"
11/29/2022 20:07:25 User requested the list of Trades on "Tuesday, November 29, 2022" at "7:07:25 PM"
11/29/2022 20:09:43 Application Starting Up on "Tuesday, November 29, 2022" at "7:09:43 PM"
11/29/2022 20:09:57 User requested the list of Trades on "Tuesday, November 29, 2022" at "7:09:57 PM"
11/29/2022 20:10:10 User requested the Trade with Id = 1 on "Tuesday, November 29, 2022" at "7:10:10 PM"
11/29/2022 20:10:17 User requested the Trade with Id = 10 on "Tuesday, November 29, 2022" at "7:10:17 PM"
11/29/2022 20:10:17 No Trade with Id = 10 was found. User was advised to enter a valid Id.
11/29/2022 20:12:40 User created a new Trade on "Tuesday, November 29, 2022" at "7:12:40 PM"
11/29/2022 20:18:20 Application Starting Up on "Tuesday, November 29, 2022" at "7:18:20 PM"
11/29/2022 20:18:53 User created a new Trade on "Tuesday, November 29, 2022" at "7:18:53 PM"
11/29/2022 20:20:44 User created a new Trade on "Tuesday, November 29, 2022" at "7:20:44 PM"
11/29/2022 20:22:33 User created a new Trade on "Tuesday, November 29, 2022" at "7:22:33 PM"
11/29/2022 20:23:09 User requested the list of Trades on "Tuesday, November 29, 2022" at "7:23:09 PM"
11/29/2022 20:24:44 User updated the Trade with Id = 1011 on "Tuesday, November 29, 2022" at "7:24:44 PM"
11/29/2022 20:25:14 User requested to delete the Trade with Id = 1015 on "Tuesday, November 29, 2022" at "7:25:14 PM"
11/29/2022 20:25:14 No Trade with Id = 1015 was found. User was advised to enter a valid Id.
11/29/2022 20:25:20 User deleted the Trade with Id = 1009 on "Tuesday, November 29, 2022" at "7:25:20 PM"
11/29/2022 20:25:28 User deleted the Trade with Id = 1010 on "Tuesday, November 29, 2022" at "7:25:28 PM"
11/29/2022 20:32:18 Application Starting Up on "Tuesday, November 29, 2022" at "7:32:18 PM"
11/29/2022 20:34:13 User created a new Trade on "Tuesday, November 29, 2022" at "7:34:13 PM"
12/01/2022 20:03:57 Application Starting Up on "Thursday, December 1, 2022" at "7:03:57 PM"
12/01/2022 20:04:17 User requested the list of Users on "Thursday, December 1, 2022" at "7:04:17 PM"
12/01/2022 20:04:32 HTTP "GET" "/user/1" responded 500 in 10.1487 ms
Microsoft.AspNetCore.Routing.Matching.AmbiguousMatchException: The request matched multiple endpoints. Matches: 

Dot.Net.WebApi.Controllers.UserController.GetSpecificUser (WebApi3)
Dot.Net.WebApi.Controllers.UserController.GetSpecificUserByUserName (WebApi3)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ReportAmbiguity(CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ProcessFinalCandidates(HttpContext httpContext, CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.Select(HttpContext httpContext, CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DfaMatcher.MatchAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.Matching.DataSourceDependentMatcher.MatchAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
12/01/2022 20:04:33 An unhandled exception has occurred while executing the request.
Microsoft.AspNetCore.Routing.Matching.AmbiguousMatchException: The request matched multiple endpoints. Matches: 

Dot.Net.WebApi.Controllers.UserController.GetSpecificUser (WebApi3)
Dot.Net.WebApi.Controllers.UserController.GetSpecificUserByUserName (WebApi3)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ReportAmbiguity(CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ProcessFinalCandidates(HttpContext httpContext, CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.Select(HttpContext httpContext, CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DfaMatcher.MatchAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.Matching.DataSourceDependentMatcher.MatchAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
12/01/2022 20:06:04 Application Starting Up on "Thursday, December 1, 2022" at "7:06:04 PM"
12/01/2022 20:06:14 User requested the User with Id = 1 on "Thursday, December 1, 2022" at "7:06:14 PM"
12/01/2022 20:06:20 User requested the User with Id = 5 on "Thursday, December 1, 2022" at "7:06:20 PM"
12/01/2022 20:06:20 No User with Id = 5 was found. User was advised to enter a valid Id.
12/01/2022 20:06:33 User requested the list of Users on "Thursday, December 1, 2022" at "7:06:33 PM"
12/01/2022 20:06:51 User requested the User with UserName = "{UserName}" on "Thursday, December 1, 2022" at "7:06:51 PM"
12/01/2022 20:06:51 No User with UserName = "{UserName}" was found. User was advised to enter a valid UserName.
12/01/2022 20:06:58 User requested the User with UserName = "{UserName}" on "Thursday, December 1, 2022" at "7:06:58 PM"
12/01/2022 20:06:58 No User with UserName = "{UserName}" was found. User was advised to enter a valid UserName.
12/01/2022 20:07:11 User requested the User with UserName = "{UserName}" on "Thursday, December 1, 2022" at "7:07:11 PM"
12/01/2022 20:07:11 No User with UserName = "{UserName}" was found. User was advised to enter a valid UserName.
12/01/2022 20:08:18 User requested the User with UserName = "{UserName}" on "Thursday, December 1, 2022" at "7:08:18 PM"
12/01/2022 20:08:18 No User with UserName = "{UserName}" was found. User was advised to enter a valid UserName.
12/02/2022 09:23:36 Application Starting Up on "Friday, December 2, 2022" at "8:23:36 AM"
12/02/2022 09:24:21 User requested the User with UserName "{UserName}" on "Friday, December 2, 2022" at "8:24:21 AM"
12/02/2022 09:24:21 No User with UserName "{UserName}" was found. User was advised to enter a valid UserName.
12/02/2022 09:25:21 User requested the User with Id = 1 on "Friday, December 2, 2022" at "8:25:21 AM"
12/02/2022 09:28:06 User requested the User with UserName "{UserName}" on "Friday, December 2, 2022" at "8:28:06 AM"
12/02/2022 09:29:04 User requested the User with Id = 2 on "Friday, December 2, 2022" at "8:29:04 AM"
12/02/2022 09:29:07 No User with UserName "{UserName}" was found. User was advised to enter a valid UserName.
12/02/2022 09:31:04 Application Starting Up on "Friday, December 2, 2022" at "8:31:04 AM"
12/02/2022 09:31:52 User requested the User with UserName null on "Friday, December 2, 2022" at "8:31:52 AM"
12/02/2022 09:32:13 No User with UserName null was found. User was advised to enter a valid UserName.
12/02/2022 09:32:40 User requested the User with UserName "user" on "Friday, December 2, 2022" at "8:32:40 AM"
12/02/2022 09:35:14 User requested the User with UserName "user" on "Friday, December 2, 2022" at "8:35:14 AM"
12/02/2022 09:36:18 Application Starting Up on "Friday, December 2, 2022" at "8:36:18 AM"
12/02/2022 09:36:54 User requested the User with UserName "{UserName}" on "Friday, December 2, 2022" at "8:36:54 AM"
12/02/2022 09:36:59 No User with UserName "{UserName}" was found. User was advised to enter a valid UserName.
12/02/2022 09:37:47 Application Starting Up on "Friday, December 2, 2022" at "8:37:47 AM"
12/02/2022 09:38:00 HTTP "GET" "/user/{UserName}" responded 500 in 8.8285 ms
Microsoft.AspNetCore.Routing.Matching.AmbiguousMatchException: The request matched multiple endpoints. Matches: 

Dot.Net.WebApi.Controllers.UserController.GetSpecificUser (WebApi3)
Dot.Net.WebApi.Controllers.UserController.GetSpecificUserByUserName (WebApi3)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ReportAmbiguity(CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ProcessFinalCandidates(HttpContext httpContext, CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.Select(HttpContext httpContext, CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DfaMatcher.MatchAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.Matching.DataSourceDependentMatcher.MatchAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
12/02/2022 09:38:00 An unhandled exception has occurred while executing the request.
Microsoft.AspNetCore.Routing.Matching.AmbiguousMatchException: The request matched multiple endpoints. Matches: 

Dot.Net.WebApi.Controllers.UserController.GetSpecificUser (WebApi3)
Dot.Net.WebApi.Controllers.UserController.GetSpecificUserByUserName (WebApi3)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ReportAmbiguity(CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.ProcessFinalCandidates(HttpContext httpContext, CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DefaultEndpointSelector.Select(HttpContext httpContext, CandidateState[] candidateState)
   at Microsoft.AspNetCore.Routing.Matching.DfaMatcher.MatchAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.Matching.DataSourceDependentMatcher.MatchAsync(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
12/02/2022 09:39:27 Application Starting Up on "Friday, December 2, 2022" at "8:39:27 AM"
12/02/2022 09:39:59 User requested the User with UserName "{UserName}" on "Friday, December 2, 2022" at "8:39:59 AM"
12/02/2022 09:40:06 No User with UserName "{UserName}" was found. User was advised to enter a valid UserName.
12/02/2022 09:41:27 Application Starting Up on "Friday, December 2, 2022" at "8:41:27 AM"
12/02/2022 09:42:01 User requested the User with UserName null on "Friday, December 2, 2022" at "8:42:01 AM"
12/02/2022 09:42:02 No User with UserName null was found. User was advised to enter a valid UserName.
12/02/2022 10:09:31 Application Starting Up on "Friday, December 2, 2022" at "9:09:31 AM"
12/02/2022 10:10:03 User requested the User with UserName "admin" on "Friday, December 2, 2022" at "9:10:03 AM"
12/02/2022 10:10:48 User requested the User with UserName "admin" on "Friday, December 2, 2022" at "9:10:48 AM"
12/02/2022 10:14:09 Application Starting Up on "Friday, December 2, 2022" at "9:14:09 AM"
12/02/2022 10:14:44 User requested the User with UserName "admin" on "Friday, December 2, 2022" at "9:14:44 AM"
12/02/2022 10:33:46 User requested the User with UserName null on "Friday, December 2, 2022" at "9:33:46 AM"
12/02/2022 10:33:58 No User with UserName null was found. User was advised to enter a valid UserName.
12/02/2022 10:37:31 User requested the list of Users on "Friday, December 2, 2022" at "9:37:31 AM"
12/02/2022 10:37:54 User requested the User with Id = 2 on "Friday, December 2, 2022" at "9:37:54 AM"
12/02/2022 10:38:12 User requested the User with Id = 5 on "Friday, December 2, 2022" at "9:38:12 AM"
12/02/2022 10:38:13 No User with Id = 5 was found. User was advised to enter a valid Id.
12/02/2022 10:38:43 User requested the User with UserName "Test" on "Friday, December 2, 2022" at "9:38:43 AM"
12/02/2022 10:39:04 User requested the User with UserName "toto" on "Friday, December 2, 2022" at "9:39:04 AM"
12/02/2022 10:39:07 No User with UserName "toto" was found. User was advised to enter a valid UserName.
12/02/2022 10:41:34 User created a new User on "Friday, December 2, 2022" at "9:41:34 AM"
12/02/2022 10:41:54 User created a new User on "Friday, December 2, 2022" at "9:41:54 AM"
12/02/2022 10:45:44 User created a new User on "Friday, December 2, 2022" at "9:45:44 AM"
12/02/2022 10:47:18 Application Starting Up on "Friday, December 2, 2022" at "9:47:18 AM"
12/02/2022 10:49:12 User created a new User on "Friday, December 2, 2022" at "9:49:12 AM"
12/02/2022 10:50:17 User requested the User with UserName "tata" on "Friday, December 2, 2022" at "9:50:17 AM"
12/02/2022 10:50:33 User requested the User with UserName "toto" on "Friday, December 2, 2022" at "9:50:33 AM"
12/02/2022 10:50:35 No User with UserName "toto" was found. User was advised to enter a valid UserName.
12/02/2022 10:50:46 User requested the User with UserName null on "Friday, December 2, 2022" at "9:50:46 AM"
12/02/2022 10:51:24 User deleted the User with Id = 1005 on "Friday, December 2, 2022" at "9:51:24 AM"
12/02/2022 10:51:43 User requested to delete the User with Id = 1010 on "Friday, December 2, 2022" at "9:51:43 AM"
12/02/2022 10:51:43 No User with Id = 1010 was found. User was advised to enter a valid Id.
12/02/2022 10:52:01 User deleted the User with Id = 1003 on "Friday, December 2, 2022" at "9:52:01 AM"
12/02/2022 10:52:14 User requested the User with UserName null on "Friday, December 2, 2022" at "9:52:14 AM"
12/02/2022 10:52:15 No User with UserName null was found. User was advised to enter a valid UserName.
12/02/2022 10:52:56 Application Starting Up on "Friday, December 2, 2022" at "9:52:56 AM"
12/02/2022 10:53:17 User requested the User with UserName "{UserName}" on "Friday, December 2, 2022" at "9:53:17 AM"
12/02/2022 10:53:21 No User with UserName "{UserName}" was found. User was advised to enter a valid UserName.
12/02/2022 10:53:44 Application Starting Up on "Friday, December 2, 2022" at "9:53:44 AM"
12/02/2022 10:54:09 Application Starting Up on "Friday, December 2, 2022" at "9:54:09 AM"
12/02/2022 10:54:31 User requested the User with UserName "Test" on "Friday, December 2, 2022" at "9:54:31 AM"
12/02/2022 10:54:46 User requested the list of Users on "Friday, December 2, 2022" at "9:54:46 AM"
12/02/2022 10:56:14 User updated the User with Id = 1004 on "Friday, December 2, 2022" at "9:56:14 AM"
12/02/2022 10:56:29 User requested the list of Users on "Friday, December 2, 2022" at "9:56:29 AM"
12/02/2022 11:42:54 Application Starting Up on "Friday, December 2, 2022" at "10:42:54 AM"
12/02/2022 11:53:19 Application Starting Up on "Friday, December 2, 2022" at "10:53:19 AM"
12/02/2022 11:54:31 User created a new User on "Friday, December 2, 2022" at "10:54:31 AM"
12/02/2022 12:00:17 Application Starting Up on "Friday, December 2, 2022" at "11:00:17 AM"
12/02/2022 12:02:36 Application Starting Up on "Friday, December 2, 2022" at "11:02:36 AM"
12/02/2022 12:03:37 Application Starting Up on "Friday, December 2, 2022" at "11:03:37 AM"
12/02/2022 12:13:29 Application Starting Up on "Friday, December 2, 2022" at "11:13:29 AM"
12/02/2022 12:14:38 User created a new User on "Friday, December 2, 2022" at "11:14:38 AM"
12/02/2022 12:17:36 Application Starting Up on "Friday, December 2, 2022" at "11:17:36 AM"
12/02/2022 12:27:17 Application Starting Up on "Friday, December 2, 2022" at "11:27:17 AM"
12/02/2022 14:53:45 Application Starting Up on "Friday, December 2, 2022" at "1:53:45 PM"
12/02/2022 14:54:26 User requested the User with Id = 1 on "Friday, December 2, 2022" at "1:54:26 PM"
12/02/2022 14:54:35 User requested the User with Id = 10 on "Friday, December 2, 2022" at "1:54:35 PM"
12/02/2022 14:54:35 No User with Id = 10 was found. User was advised to enter a valid Id.
12/02/2022 14:55:22 User requested the User with UserName "user" on "Friday, December 2, 2022" at "1:55:22 PM"
12/02/2022 14:55:28 User requested the User with UserName null on "Friday, December 2, 2022" at "1:55:28 PM"
12/02/2022 14:55:28 No User with UserName null was found. User was advised to enter a valid UserName.
12/02/2022 14:57:21 Application Starting Up on "Friday, December 2, 2022" at "1:57:21 PM"
12/02/2022 14:57:46 User requested the User with UserName "user" on "Friday, December 2, 2022" at "1:57:46 PM"
12/02/2022 14:58:45 Application Starting Up on "Friday, December 2, 2022" at "1:58:45 PM"
12/02/2022 17:24:11 Application Starting Up on "Friday, December 2, 2022" at "4:24:11 PM"
12/02/2022 17:24:59 HTTP "POST" "/login" responded 500 in 25.6299 ms
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
12/02/2022 17:24:59 An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
12/02/2022 17:26:31 Application Starting Up on "Friday, December 2, 2022" at "4:26:31 PM"
12/02/2022 17:26:54 HTTP "POST" "/login" responded 500 in 26.8983 ms
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
12/02/2022 17:26:54 An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
12/02/2022 17:29:21 Application Starting Up on "Friday, December 2, 2022" at "4:29:21 PM"
12/02/2022 17:29:37 HTTP "POST" "/login" responded 500 in 24.6720 ms
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
12/02/2022 17:29:37 An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
12/02/2022 17:30:57 HTTP "POST" "/login" responded 500 in 6.6447 ms
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
12/02/2022 17:30:57 An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
12/02/2022 17:31:08 Application Starting Up on "Friday, December 2, 2022" at "4:31:08 PM"
12/02/2022 17:31:23 HTTP "POST" "/login" responded 500 in 24.6823 ms
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
12/02/2022 17:31:23 An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
12/02/2022 17:32:39 Application Starting Up on "Friday, December 2, 2022" at "4:32:39 PM"
12/02/2022 17:32:58 HTTP "POST" "/login" responded 500 in 25.5511 ms
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
12/02/2022 17:32:58 An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
12/02/2022 17:38:27 Application Starting Up on "Friday, December 2, 2022" at "4:38:27 PM"
12/02/2022 17:38:39 HTTP "POST" "/login" responded 500 in 27.2690 ms
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
12/02/2022 17:38:39 An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
12/02/2022 17:41:22 Application Starting Up on "Friday, December 2, 2022" at "4:41:22 PM"
12/02/2022 17:41:34 HTTP "POST" "/login" responded 500 in 24.7028 ms
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
12/02/2022 17:41:34 An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
12/02/2022 17:43:32 Application Starting Up on "Friday, December 2, 2022" at "4:43:32 PM"
12/02/2022 17:43:49 HTTP "POST" "/login" responded 500 in 26.3922 ms
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
12/02/2022 17:43:49 An unhandled exception has occurred while executing the request.
System.InvalidOperationException: Unable to resolve service for type 'WebApi3.Repositories.LoginRepository' while attempting to activate 'Dot.Net.WebApi.Controllers.LoginController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
12/02/2022 17:44:43 Application Starting Up on "Friday, December 2, 2022" at "4:44:43 PM"
12/02/2022 17:50:06 Application Starting Up on "Friday, December 2, 2022" at "4:50:06 PM"
12/02/2022 17:58:34 Application Starting Up on "Friday, December 2, 2022" at "4:58:34 PM"
12/02/2022 17:59:59 Application Starting Up on "Friday, December 2, 2022" at "4:59:59 PM"
12/02/2022 18:00:36 Application Starting Up on "Friday, December 2, 2022" at "5:00:36 PM"
12/02/2022 18:21:40 Application Starting Up on "Friday, December 2, 2022" at "5:21:40 PM"
12/02/2022 18:24:53 User created a new User on "Friday, December 2, 2022" at "5:24:53 PM"
12/02/2022 18:39:21 Application Starting Up on "Friday, December 2, 2022" at "5:39:21 PM"
